test/test-utils.js

```diff
@@ -66,6 +66,37 @@ describe('utils', function() {
     });
   });
 
+  describe('getProjectId', function() {
+
+    it('should be able to get project number from metadata service',
```

Comment:
Maybe add a test resulting in an error response or status code. I appreciate `getProjectNumber` doesn't have such a test but I think it would be a good addition.